public/app/plugins/panel/table/column_options.ts

```diff
@@ -44,6 +45,14 @@ export class ColumnOptionsCtrl {
       { text: 'YYYY-MM-DD', value: 'YYYY-MM-DD' },
     ];
     this.mappingTypes = [{ text: 'Value to text', value: 1 }, { text: 'Range to text', value: 2 }];
+    this.alignTypes = [
+      { text: 'default', value: '' },
+      { text: 'left', value: 'left' },
+      { text: 'center', value: 'center' },
+      { text: 'right', value: 'right' },
+      { text: 'justify', value: 'justify' },
+      { text: 'initial', value: 'initial' },
```

Comment:
Not sure if `initial` is something we want in this context? Shouldn't default cover this?

Reply:
Just added all the css styles there are.

> One last thing, for existing dashboards/table panels the align select/drop down doesn't seem to work but for new table panels it works:
> ![table-align](https://user-images.githubusercontent.com/562238/71093324-42ac5000-21a9-11ea-85fe-a6e3628b5414.gif)

Had (and still have) no idea how to make it proper.